content/scaling-kubernetes-cluster/scale-pods-horizontally.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Scale Kubernetes pods horizontally"
+description: "Procedures to scale Kubernetes pods horizontally by modifying the replica count"
+weight: 3
+---
+
```

Comment:
This makes no reference of `HorizontalPodAutoscalers`

Reply:
https://github.com/kubernetes/kops/blob/master/docs/horizontal_pod_autoscaling.md